lib/util/esWrapper.js

```diff
@@ -114,63 +121,106 @@ class ESWrapper {
     }
 
     if (error instanceof es.errors.NoConnections) {
-      return new ServiceUnavailableError('Elasticsearch service is not connected');
+      errorsManager.throw(
+        'external',
+        'elasticsearch',
+        'elasticsearch_service_not_connected');
     }
 
     let message = error.message || '';
 
-    const messageReplaced = errorMessagesMapping.some(mapping => {
-      message = message.replace(mapping.regex, mapping.replacement);
-      return message !== error.message;
-    });
+    let kuzzleError = null;
+
+    for (const mapping of errorMessagesMapping) {
+      const matches = message.match(mapping.regex, mapping.replacement);
+
+      if (!matches) {
+        continue;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this "continue" statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AContinueStatement)